packages/terraform-modules/testnet/modules/validator/startup.sh

```diff
@@ -83,14 +83,16 @@ docker run -v $DATA_DIR:$DATA_DIR --name geth --net=host --entrypoint /bin/sh -d
     echo -n '${genesis_content_base64}' | base64 -d > /var/geth/genesis.json && \
     echo -n '${rid}' > $DATA_DIR/replica_id && \
     echo -n '${ip_address}' > $DATA_DIR/ipAddress && \
-    echo -n '$PRIVATE_KEY' > $DATA_DIR/pkey && \
     echo -n '$ACCOUNT_ADDRESS' > $DATA_DIR/address && \
     echo -n '$BOOTNODE_ENODE_ADDRESS' > $DATA_DIR/bootnodeEnodeAddress && \
     echo -n '$BOOTNODE_ENODE' > $DATA_DIR/bootnodeEnode && \
     echo -n '$GETH_ACCOUNT_SECRET' > $DATA_DIR/account/accountSecret && \
     geth init /var/geth/genesis.json
   ) && ( \
-    geth account import --password $DATA_DIR/account/accountSecret $DATA_DIR/pkey ; \
+    TMP_PRIVATE_KEY_FILE=$(mktemp) \
```

Comment:
Could you put a `;` or `&&` after this? otherwise I think the following line might cause problems

Reply:
Done.